src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -375,23 +375,22 @@ def _osd_rm(self, svc_id):
     def _rbd_mirror_add(self, num=None, hosts=None):
         spec = orchestrator.StatelessServiceSpec(
             None,
-            placement=orchestrator.PlacementSpec(nodes=hosts),
-            count=num or 1)
+            placement=orchestrator.PlacementSpec(nodes=hosts, count=num))
         completion = self.add_rbd_mirror(spec)
         self._orchestrator_wait([completion])
         orchestrator.raise_if_exception(completion)
         return HandleCommandResult(stdout=completion.result_str())
 
     @orchestrator._cli_write_command(
         'orchestrator rbd-mirror update',
-        "name=num,type=CephInt,req=true "
+        "name=num,type=CephInt,req=false "
+        "name=label,type=CephString,req=false "
         "name=hosts,type=CephString,n=N,req=false",
         'Update the number of rbd-mirror instances')
-    def _rbd_mirror_update(self, num, hosts=None):
+    def _rbd_mirror_update(self, num, label=None, hosts=[]):
```

Comment:
`[]` is a mutable default argument. This works as long as no one modifies `hosts`